acceptance/localcluster/localcluster.go

```diff
@@ -564,9 +564,11 @@ func (l *Cluster) stop() {
 				log.Infof("node %d: stderr at %s", i, file)
 			}
 		} else if crashed {
+			// Don't remove crashed containers - could be useful for post-mortem.
```

Comment:
You're right. I still think that this isn't useful if it has to be explicitly turned on - that's not usually how you run into a node crash. I'm closing, but feel free to run with this if you feel stronger than I do that this is needed.